### PR TITLE
Trim leading zeros from AppX package version

### PR DESCRIPTION
**Pull request by gkhanna79**
AppX package version does not like leading zero that come from out build infrastructure. Trim them out to get the expected version.

Was able to repro locally and validate the fix.

@weshaggard PTAL.

### CODE REVIEW

**Comment by gkhanna79** [comment]
@weshaggard Are you good with this? I would like to merge this to unblock official build of master.

**Comment by weshaggard** [comment]
It seems like a hack but I don't fully understand the issue to provide an alternative. I wonder why it isn't an issue for our Assembly versions https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L24. At a minimum I would suggest pulling the version into a property to make it easier to read as that line is getting a little crazy.

**Comment by gkhanna79** [comment]
It does not matter for Assembly versions since they are not processed by MakeAppX tool that follows a schema for version in AppXManifest.

**Comment by gkhanna79** [comment]
@weshaggard Converted to properties.

File: src/pkg/projects/Microsoft.NET.CoreRuntime/Microsoft.NET.CoreRuntime.depproj [hunk 1]
```diff
@@ -14,6 +14,9 @@
     <AppxIdentityName>Microsoft.NET.CoreRuntime.$(MajorVersion).$(MinorVersion)</AppxIdentityName>
     <AppxDisplayName>Microsoft.NET.CoreRuntime.$(MajorVersion).$(MinorVersion)</AppxDisplayName>
     
+    <BuildVersionAppX>$([System.Int32]::Parse($(BuildNumberMajor)))</BuildVersionAppX>
+    <RevisionVersionAppX>$([System.Int32]::Parse($(BuildNumberMinor)))</RevisionVersionAppX>
```
Comment (weshaggard) [inline src/pkg/projects/Microsoft.NET.CoreRuntime/Microsoft.NET.CoreRuntime.depproj]: I think I would also move AppxVersion to be a property defined as `$(MajorVersion).$(MinorVersion).$(BuildVersionAppX).$(RevisionVersionAppX))` to keep the Replace statement a simple single property map.

**Pull request merged**